### PR TITLE
Add missing statistics data when using SQL storage (#6026)

### PR DESCRIPTION
**Pull request by jamesharling**
Fixes https://github.com/NuGet/NuGetGallery/issues/6026

### CODE REVIEW

File: src/NuGetGallery/Services/SqlAggregateStatsService.cs [hunk 1]
```diff
@@ -13,9 +13,10 @@ public class SqlAggregateStatsService : IAggregateStatsService
 
         // Note the NOLOCK hints here!
         private static readonly string GetStatisticsSql = @"SELECT
-                    (SELECT COUNT([Key]) FROM PackageRegistrations pr WITH (NOLOCK)
-                            WHERE EXISTS (SELECT 1 FROM Packages p WITH (NOLOCK) WHERE p.PackageRegistrationKey = pr.[Key] AND p.Listed = 1 AND p.PackageDelete_Key IS NULL)) AS UniquePackages,
-                    (SELECT COUNT([Key]) FROM Packages WITH (NOLOCK) WHERE Listed = 1) AS TotalPackages";
+    (SELECT SUM([DownloadCount]) FROM PackageRegistrations WITH (NOLOCK)) As Downloads,
+    (SELECT COUNT([Key]) FROM PackageRegistrations pr WITH (NOLOCK)
+            WHERE EXISTS (SELECT 1 FROM Packages p WITH (NOLOCK) WHERE p.PackageRegistrationKey = pr.[Key] AND p.Listed = 1 AND p.PackageDelete_Key IS NULL)) AS UniquePackages,
```
Comment (loic-sharma) [inline src/NuGetGallery/Services/SqlAggregateStatsService.cs]: `p.PackageDelete_Key IS NULL` should be `p.PackageStatusKey = 0` to support galleries that have asynchronous validations (see [this](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery.Core/PackageStatus.cs#L11) for more information).

**Comment by jamesharling** [comment]
Does this need any changes re your review? `p.PackageDelete_Key IS NULL` was in the latest `dev` branch code, only the `Downloads` property was added by me.

**Comment by loic-sharma** [comment]
The `p.PackageDelete_Key IS NULL` was outdated, we've been changing everything to use the `PackageStatusKey`. It looks like we forgot to update the `SqlAggregateStatsService`. If you could fix that as part of your change, that'd be great!

**Comment by jamesharling** [comment]
@loic-sharma SQL adjusted as per request

**Comment by loic-sharma** [comment]
Merged! Thank you @jamesharling

**Pull request merged**